### PR TITLE
Added new podcasts to english languages

### PR DESCRIPTION
**Pull request by Ratheshan03**
## What does this PR do?
Added new Podcasts in English language
 - DevOps new section added 
 - Added more Swift podcasts

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: casts/free-podcasts-screencasts-en.md [hunk 1]
```diff
@@ -78,6 +80,11 @@
 * [The Cloud Pod](https://www.thecloudpod.net) - Justin Brodley, Jonathan Baker, Ryan Lucas and Peter Roosakos (podcast)
 
 
+### Data Engineering
+
+* [Data Engineering Podcast](https://www.dataengineeringpodcast.com) - Tobias Macey (podcast)
```
Comment (eshellman) [inline casts/free-podcasts-screencasts-en.md:85]: just put it under data science

**Review by eshellman — CHANGES_REQUESTED** [review]

File: casts/free-podcasts-screencasts-en.md (hunk 1 above)
Comment (Ratheshan03) [reply casts/free-podcasts-screencasts-en.md:85]: okay sure!

**Comment by Ratheshan03** [comment]
@eshellman I made the changes 😄

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**